_scripts/mime-db-shrinking-loader.js

```diff
@@ -0,0 +1,26 @@
+/**
+ * electron-context-menu only needs mime-db for it's save as feature.
```

Comment:
```suggestion
 * electron-context-menu only needs mime-db for its save as feature.
```

This is very pedantic of me, I'm very sorry lol